modular_zubbers/code/modules/clothing/suits/misc.dm

```diff
@@ -53,3 +53,26 @@
 		/obj/item/radio,
 		/obj/item/storage/belt/holster,
 	)
+
+/obj/item/clothing/suit/misc/syncapcarapace/fake
```

Comment:
This is a child object but it appears you copied all the vars from the parent (which already would trickle down) without nulling the vars that actually matter (like armour)